src/Mizar/CMakeLists.txt

```diff
@@ -4,22 +4,38 @@
 
 cmake_minimum_required(VERSION 3.15)
 
-project(Mizar CXX) 
+project(Mizar CXX)
 
 add_executable(Mizar)
 
 target_sources(
-  Mizar
-  PRIVATE Mizar.cpp)
+        Mizar
+        PRIVATE Mizar.cpp
+        MizarMainWindow.h
+        MizarMainWindow.cpp
+        MizarMainWindow.ui
+        SamplingWithFrameTrackInputWidget.h
+        SamplingWithFrameTrackInputWidget.cpp
+        SamplingWithFrameTrackInputWidget.ui
+        SamplingWithFrameTrackWidget.h
+        SamplingWithFrameTrackWidget.cpp
+        SamplingWithFrameTrackWidget.ui)
 
-target_include_directories(Mizar PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}) 
+target_include_directories(Mizar PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
+target_include_directories(Mizar PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include)
 
-target_link_libraries(Mizar 
-PRIVATE CaptureClient
+target_link_libraries(Mizar
+        PUBLIC QtUtils
+        Qt5::Widgets
+        Qt5::Core
+        PRIVATE CaptureClient
```

Comment:
Since Mizar is an executable target, there is no difference between PRIVATE and PUBLIC. Imho all dependencies should be marked as PRIVATE